testing/testcafe/helpers/themeUtils.ts

```diff
@@ -0,0 +1,40 @@
+import { isString } from '../../../js/core/utils/type';
+import { changeTheme } from './changeTheme';
+
+export const getThemePostfix = (theme?: string): string => {
+  const themeName = (theme ?? process.env.theme) ?? 'generic.light';
+  return `-theme=${themeName.replace(/\./g, '-')}`;
```

Comment:
screenshots' names looks strange to me now. Maybe make it like this?
```suggestion
  return ` (theme=${themeName.replace(/\./g, '-')})`;
```

Reply:
Let's use this theme postfix ` (${themeName.replace(/\./g, '-')})`